pyQuARC/code/schema_validator.py

```diff
@@ -111,12 +64,18 @@ def _build_errors(error_log, paths):
         errors = {}
         lines = error_log.splitlines()
         for line in lines:
-            field_name = re.search("Element\s\'(\w+)\'", line)[1]
+            # For DIF, because the namespace is specified in the metadata file, lxml library
+            # provides field name concatenated with the namespace,
+            # the following 3 lines of code removes the namespace
+            namespaces = re.findall("(\{http[^}]*\})", line)
+            for namespace in namespaces:
+                line = line.replace(namespace, '')
+            field_name = re.search("Element\s\'(.*)\':", line)[1]
             field_paths = [
                 abs_path for abs_path in paths if field_name in abs_path
             ]
             field_name = field_paths[0] if len(field_paths) == 1 else field_name
-            message = re.search("Element\s\'\w+\':\s(\[.*\])?(.*)", line)[2].strip()
+            message = re.search("Element\s\'.+\':\s(\[.*\])?(.*)", line)[2].strip()
```

Comment:
Any reason for the change?

Reply:
i don't remember exactly what, but it wasn't matching one particular condition